### DISCUSSION TITLE
Capture `fig-alt=` from the first image for use in the listing page

### DISCUSSION THREAD

**Discussion by cgoo4** [quarto-dev/quarto-cli#3704] [Feature Requests]
When `image:` is left unspecified in a `.qmd` YAML header, the first image found is also used for the listing page. This is great as it avoids specifying the image twice when it's wanted in both the listing and as the feature image for the post.

Could the alternative text from the first image also flow through to the listing page? It would avoid the need to specify this twice, i.e. in the YAML as `image-alt:` and in the inserted image as `fig-alt=`.

Example where specified twice to appear in both the listing and the post: [post in repo](https://github.com/cgoo4/quantumjitter/blob/main/blog/dark/index.qmd)
Example where specified in the first image only and doesn't flow to the listing with the image: [post in repo](https://github.com/cgoo4/quantumjitter/blob/main/project/stories/index.qmd)

**Comment by dragonstyle** [comment]
This is a great suggestion- thank you! I'm converting to an issue to track it.

**Comment by mcanouil** [comment]
Closing as completed and in favour of https://github.com/quarto-dev/quarto-cli/issues/3706
